mlflow/server/js/src/experiment-tracking/pages/prompts/hooks/useEditModelConfigModal.tsx

```diff
@@ -0,0 +1,141 @@
+/**
```

Comment:
We don't have doc string for component usage, so can we remove this section for consistency?